Kickstarter-iOS/Views/Controllers/UpdatePreviewViewController.swift

```diff
@@ -4,51 +4,51 @@ import Prelude
 import UIKit
 
 internal final class UpdatePreviewViewController: WebViewController {
-  private let viewModel: UpdatePreviewViewModelType = UpdatePreviewViewModel()
+  fileprivate let viewModel: UpdatePreviewViewModelType = UpdatePreviewViewModel()
 
-  @IBOutlet private weak var publishBarButtonItem: UIBarButtonItem!
+  @IBOutlet fileprivate weak var publishBarButtonItem: UIBarButtonItem!
 
-  internal static func configuredWith(draft draft: UpdateDraft) -> UpdatePreviewViewController {
-    let vc = Storyboard.UpdateDraft.instantiate(UpdatePreviewViewController)
+  internal static func configuredWith(draft: UpdateDraft) -> UpdatePreviewViewController {
+    let vc = Storyboard.UpdateDraft.instantiate(UpdatePreviewViewController.self)
     vc.viewModel.inputs.configureWith(draft: draft)
     return vc
   }
 
   internal override func viewDidLoad() {
     super.viewDidLoad()
 
-    self.publishBarButtonItem
+    _ = self.publishBarButtonItem
       |> UIBarButtonItem.lens.targetAction .~ (self, #selector(publishButtonTapped))
 
     self.viewModel.inputs.viewDidLoad()
   }
 
   internal override func bindStyles() {
-    self |> baseControllerStyle()
-    self.publishBarButtonItem |> updatePreviewBarButtonItemStyle
+    _ = self |> baseControllerStyle()
+    _ = self.publishBarButtonItem |> updatePreviewBarButtonItemStyle
 
     self.navigationItem.title = nil
   }
 
   internal override func bindViewModel() {
     self.viewModel.outputs.webViewLoadRequest
       .observeForControllerAction()
-      .observeNext { [weak self] in self?.webView.loadRequest($0) }
+      .observeValues { [weak self] in _ = self?.webView.load($0) }
```

Comment:
again, not a big deal, but might be more readable with the assignment on a second line.